### PR TITLE
[Wiki] Updated template-orchestrated solution examples

### PR DESCRIPTION
**Pull request by MrMCake**
# Description

- Updated template-orchestrated solution examples
  - Moved parameters into module deployments
  - Simplified references

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Update to documentation

### CODE REVIEW

File: docs/wiki/Solution creation.md [hunk 1]
```diff
@@ -267,28 +225,40 @@ module rg 'br/modules:microsoft.resources.resourcegroups:0.4.735' = {
...
+        networkSecurityGroupName: nsg.name
+      }
+      {
+        name: 'SecondarySubnet'
+        addressPrefix: '10.0.1.0/24'
+        networkSecurityGroupName: nsg.name
+      }
+    ]
   }
   dependsOn: [
```
Comment (eriqua) [inline docs/wiki/Solution creation.md:261]: Isn't this indirectly achieved by subnets depending on nsg which in turns depends on rg?

Comment (MrMCake) [reply docs/wiki/Solution creation.md:261]: Yes., but it would be transient and hence potentially harder to understand. Thoughts?

Comment (eriqua) [reply docs/wiki/Solution creation.md:261]: I'd avoid code which is not necessary, but I don't find that hard to understand so maybe my opinion is biased. Besides that I don't have other thoughts

Comment (MrMCake) [reply docs/wiki/Solution creation.md]: don't have a strong opinion about it - and you at least have an opinion, so I updated it now :D

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**